src/Illuminate/Testing/composer.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "illuminate/testing",
+    "description": "The Illuminate Testing package.",
+    "license": "MIT",
+    "homepage": "https://laravel.com",
+    "support": {
+        "issues": "https://github.com/laravel/framework/issues",
+        "source": "https://github.com/laravel/framework"
+    },
+    "authors": [
+        {
+            "name": "Taylor Otwell",
+            "email": "taylor@laravel.com"
+        }
+    ],
+    "require": {
+        "php": "^7.2.5",
+        "mockery/mockery": "^1.3.1",
+        "phpunit/phpunit": "^8.4|^9.0"
+    },
+    "autoload": {
+        "psr-4": {
+            "Illuminate\\Testing\\": ""
+        }
+    },
+    "extra": {
+        "branch-alias": {
+            "dev-master": "7.0-dev"
```

Comment:
This needs to be 
```suggestion
            "dev-master": "7.x-dev"
```

Reply:
I copy paste the value from `translation` composer.json

https://github.com/laravel/framework/blob/master/src/Illuminate/Translation/composer.json#L28-L31

Reply:
Thanks for making me aware. I've fixed these on master 👍